lit_nlp/examples/gcp/README.md

```diff
@@ -0,0 +1,32 @@
+# Using LLMs in LIT on Google Cloud Platform
+
+## Developing
+
+### Use a virtual environment
+
+```shell
+# Create and activate the virtual environment
+python3 -m venv ~/.venvs/lit-on-gcp
+source ~/.venvs/lit-on-gcp/bin/activate
+
+# Install the requirements and LIT in editable mode
+pip install -f ./lit_nlp/examples/gcp/requirements.txt
+pip install -e .
+
+# Optionally, install tetsing requirements
+pip install -f ./requirements_test.txt
+pytest pytest lit_nlp/examples/gcp
+```
+
+### Build the Docker image
+
+```shell
+docker build -f ./lit_nlp/examples/gcp/Dockerfile -t lit-app:gcp-dev .
+```
+
+### Run GPT-2 in a Docker container
+
+```shell
+# Runs GPT-2 in Keras on Tensorflow
```

Comment:
Should this be pytorch, which is the default runtime now?

Reply:
Yes, good catch.